tutorials/quickstarts/README.md

```diff
@@ -0,0 +1,18 @@
+# Quickstarts for Compute Instance
+
+description: Quickstarts for use in Compute Instance
+
+The notebooks in this tutorial will be used as getting started examples on the Notebooks landing page in AML Studio, and will be run in a Compute Instance.
+
+The tutorial consists of three notebooks:
+
+- [AzureMLin10mins.ipynb](AzureMLin10mins.ipynb)
+- [GettingStartedWithPythonSDK.ipynb](GettingStartedWithPythonSDK.ipynb)
+- [ClassificationWithAutomatedML.ipynb](ClassificationWithAutomatedML.ipynb)
+
+**AzureML in 10mins** 
+This notebook takes the user through a data ingest, training and deployment workflow that can be completed in 10minutes. This notebook does *not* use job submission, but rather an interactive run to reduce the number of concepts. 
```

Comment:
add an empty line between the bold header and text

Reply:
Will do.